AWS/aws.js

```diff
@@ -883,6 +891,38 @@ const queryNeptuneClusterParameterGroups = async (
   AWS_MAPPING.total += resourceCount;
 };
 
+const queryComputeOptimizerEnrollmentStatus = async (
+  serviceName,
+  resourceType,
+  region
+) => {
+  let total = 0;
+  const client = new ComputeOptimizerClient({ region });
+  const resources = [];
+  const command = new GetEnrollmentStatusCommand({});
+  const response = await client.send(command);
+  resources.push({ ...(response || {}) });
+  updateResourceTypeCounter(serviceName, resourceType, resources.length);
+  total += resources.length;
+  AWS_MAPPING.total += total;
+};
+
+const queryScalingPolicies = async (serviceName, resourceType, region) => {
```

Comment:
Do we need this code block as you're importing ScalingPolices from aws-dependencies.json

Reply:
This one is for `AWS::ApplicationAutoScaling::ScalingPolicy` and the other one is `AWS::AutoScaling::ScalingPolicy`